tests/test_transforms.py

```diff
@@ -140,6 +140,19 @@ def test_normalize(input, axis, expected_output):
     assert np.array_equal(t(input), expected_output)
 
 
+@pytest.mark.parametrize('input,axis', [
+    (A, None),
+    (A, -1),
+    (a11, None),
+    (a11, -1),
+])
+def test_standardize(input, axis):
+    t = transforms.Standardize(axis=axis)
+    output = t(input)
+    np.testing.assert_almost_equal(output.mean(axis=axis).mean(), 0)
+    np.testing.assert_almost_equal(output.std(axis=axis).mean(), 1)
```

Comment:
Please also add a test for `F.standardize` in `tests/test_transforms_functional.py`.